topics/icrc_25_signer_interaction_standard.md

```diff
@@ -38,6 +38,7 @@ The purpose of the `permission` messages is to grant the relying party access to
 
 - `text`: A plain `string` value.
 - `blob`: A `string` value describing binary data encoded in base64.
+- `int` An integer value.
```

Comment:
Can you also lift out the `Types` into it's own section (i.e. `## Types`). I don't think it makes sense to define that multiple times.